contracts/apm/APMRegistry.sol

```diff
@@ -8,25 +8,25 @@ import "../acl/ACL.sol";
 import "./Repo.sol";
 
 
-contract APMRegistryConstants {
-    // Cant have a regular APM appId because it is used to build APM
```

Comment:
AFAIK these apps are all registered in [`APMRegistryFactory` as having normal `<name>.aragonpm.eth` names now](https://github.com/aragon/aragonOS/blob/dev/contracts/factory/APMRegistryFactory.sol#L102)